Makefile

```diff
@@ -52,13 +52,17 @@ test-snowflake-only:
 	cd kedro-datasets && pytest --no-cov --numprocesses 1 --dist loadfile -m snowflake
 	cd kedro-datasets && pytest kedro_datasets/snowflake --no-cov
 
-check-datasets-docs:
-	cd kedro-datasets && python -m sphinx -WETan -j auto -D language=en -b linkcheck -d _build/doctrees docs/source _build/linkcheck
+build-datasets-docs:
+	# this checks: mkdocs.yml is valid, all listed pages exist, plugins are correctly configured, no broken references in nav or Markdown links (internal), broken links and images (internal, not external)
+	cd kedro-datasets && mkdocs build
```

Comment:
Is this needed with the Read the Docs build?

Reply:
This should eventually become `mkdocs build --strict` and will allow us to both locally and on GH actions catch any missing internal links, files etc.